pkg/volume/util/operationexecutor/operation_generator.go

```diff
@@ -1789,48 +1789,26 @@ func isDeviceOpened(deviceToDetach AttachedVolume, hostUtil mount.HostUtils) (bo
 	return deviceOpened, nil
 }
 
-func useCSIPlugin(vpm *volume.VolumePluginMgr, spec *volume.Spec) bool {
-	// TODO(#75146) Check whether the driver is installed as well so that
-	// we can throw a better error when the driver is not installed.
-	// The error should be of the approximate form:
-	// fmt.Errorf("in-tree plugin %s is migrated on node %s but driver %s is not installed", pluginName, string(nodeName), driverName)
+func useCSIPlugin(vpm *volume.VolumePluginMgr, spec *volume.Spec, nodeName *types.NodeName) bool {
 	if !utilfeature.DefaultFeatureGate.Enabled(features.CSIMigration) {
 		return false
 	}
 	if csilib.IsPVMigratable(spec.PersistentVolume) || csilib.IsInlineMigratable(spec.Volume) {
 		migratable, err := vpm.IsPluginMigratableBySpec(spec)
 		if err == nil && migratable {
-			return true
+			if nodeName == nil {
+				return true
```

Comment:
comment for when this is the case, should we be logging a warning?

Reply:
I have added a comment.

Since nodeName being nil is normal case (mount operation), there is no need to log warning.